pkg/web/static/js/main.js

```diff
@@ -43,4 +43,26 @@ document.body.addEventListener('htmx:responseError', (e) => {
       document.querySelector('.alert').remove()
     }, 5000)
   }
-})
\ No newline at end of file
+})
+
+/*
+ * Timestamp handling: localizes and formats datetimes on the page.
+ */
+const updateDatetimes = (e) => {
+  const options = {};
+  const dtfmt = Intl.DateTimeFormat(navigator.languages, {
+    year: 'numeric', month: 'long', day: 'numeric',
+    hour: 'numeric', minute:'numeric', second: 'numeric',
+    hour12: false, timeZoneName: "short",
+  });
+
+  const datetimes = document.querySelectorAll('.datetime');
+  datetimes?.forEach(elem => {
+    const dt = new Date(elem.textContent);
+    elem.textContent = dtfmt.format(dt);
+    elem.classList.remove('datetime');
+  });
+};
+
+document.addEventListener('DOMContentLoaded', updateDatetimes);
```

Comment:
Was `DOMContentLoaded` added here for when content isn't swapped? I believe an `afterSettle` event occurs whenever there's a `GET` request which should be triggered for all of the tables. We can double check though.

Reply:
Yeh, I didn't know if there were any timestamps in not HTMX areas; I didn't even know if an event handler was the right thing. Happy for your advise on how to handle this.

Reply:
We only have timestamps in the tables so all of those are swapped in with htmx. I escaped this line and the timestamps still appeared in the correct time zone. This line can be removed, but feel free to double check on your end.